pages/api/handoff.page.ts

```diff
@@ -36,13 +37,17 @@ export const returnRedirectUrl = async (
     >({
       query: GetDefaultAccountDocument,
     });
-    let defaultAccountID = req.query?.accountListId;
-    if (!defaultAccountID) {
```

Comment:
This is what was causing the error. We were passing the Impersonatee's accountListId to the old MPDx causing the error.

Reply:
If the user is impersonating, won't `user` be the impersonated user (not the logged in user) since we are using `impersonatorApiToken`? If that user has a `defaultAccountList`, I believe this will still use the impersonated user's account list id. But maybe I'm misunderstanding.

Reply:
No that makes sense, I was just misunderstanding